lib/challenge.js

```diff
@@ -35,7 +35,9 @@ function prompt1() {
  */
 
 function prompt2() {
-  // YOUR CODE HERE
+  for(let i=1; i<100; i++){
+    if(i%2===0)console.log(i);
```

Comment:
Best practice I would include spaces throughout for readability. So as such:
`for(let i = 1; i < 101; i++) {
     if(i % 2 === 0) console.log(i)
  }`